docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -147,6 +147,10 @@ The location of the log file will vary, depending on your operating system. For
 
 Look for the **Active Tunnel** confirmation.<br/><img src={useBaseUrl('img/sauce-connect/tunnelsuccess-ui.png')} alt="Sauce Connect Tunnel Success" width="500"/>
 
+#### Programatically Checking for Tunnel Readiness
+
+There are several ways of automatically checking for [tunnel readiness](/secure-connections/sauce-connect/setup-configuration/ci-cd-integration/readiness-checks).
```

Comment:
The 2 methods under Manual Testing (SC client and log file) in /secure-connections/sauce-connect/setup-configuration/readiness-checks/#manual-testing are already included in this section -- we could change the heading to link to the readiness check methods under CI/CD Testing.
```suggestion
#### CI/CD System

If you're starting ephemeral tunnels from a CI/CD system, there are multiple ways to automatically check for [tunnel readiness](/secure-connections/sauce-connect/setup-configuration/readiness-checks/#cicd-testing).
```